### ISSUE TITLE
Notifications of new transactions on Alby icon (badge)

### ISSUE THREAD

**Issue by bitcoinuser** [getAlby/lightning-browser-extension#1536]
### Is your feature request related to a problem? Please describe

Today, users are not notified of new transactions. They need to open Alby to see if they received new transactions

### Describe the solution you'd like

I would like to see notifications of new transactions on Alby icon (badge). Like this: https://clevertap.com/blog/app-icon-badges/

### Are you working on this?

No

**Comment by escapedcat** [comment]
You mean on the favicon?
<img width="42" alt="image" src="https://user-images.githubusercontent.com/1016218/192678531-700a33fb-a808-4deb-83f6-771d4b940d72.png">

**Comment by bitcoinuser** [comment]
> You mean on the favicon?
> 
>     * <img alt="image" width="42" src="https://user-images.githubusercontent.com/1016218/192678531-700a33fb-a808-4deb-83f6-771d4b940d72.png">
> 
>     * ![image](https://user-images.githubusercontent.com/1016218/192678591-e5af5208-1d54-4b8d-8ab0-b5af2e73ab10.png)

Yes

**Comment by reneaaron** [comment]
Generally like the idea. There are badges available to display on the extension icon: 

https://developer.chrome.com/docs/extensions/reference/browserAction/#method-setBadgeText
`browser.browserAction.setBadgeText({ text: "25" });`

![image](https://user-images.githubusercontent.com/100827540/194705023-f9eb8917-d160-45f2-82f6-3752a0b63edf.png)

Questions to tackle:
 - Fetching of transactions in the background on a regular basis? Which interval? 
 - When do we mark transactions as "seen"?

**Comment by bitcoinuser** [comment]
> * When do we mark transactions as "seen"?

I think the simple action of opening Alby and clicking on Incoming tab, should mark as seen.

**Comment by bitcoinuser** [comment]
> > * When do we mark transactions as "seen"?
> 
> I think the simple action of opening Alby and clicking on Incoming tab, should mark as seen. Maybe when there are new transactions, when clicking on Alby favicon, should go directly to Incoming tab.

Maybe on Incoming tab, have a button called (New transactions), when the user click on it expand and show the new transactions and at same time is marked as seen.

**Comment by escapedcat** [comment]
> Maybe on Incoming tab, have a link called (show new transactions), when the user click on it, expand the list and show the new transactions and at same time mark them as seen.

We could also add a "dot" to the "incoming" tab to indicate new ones which will be removed after clicking the tab. New txs could be marked as well maybe.

Regarding using the "badgeText": Do you think there's a chance we might use this for something else in the future? Once a user is used to "these are new transactions" it might not be easy to use it for something else. Just a thought.

**Comment by reneaaron** [comment]
> Regarding using the "badgeText": Do you think there's a chance we might use this for something else in the future? Once a user is used to "these are new transactions" it might not be easy to use it for something else. Just a thought.

Good point. We should put some thought into that. Right from the top of my head I couldn't come up with something else that would make sense in there though.

**Comment by bitcoinuser** [comment]
> We could also add a "dot" to the "incoming" tab to indicate new ones which will be removed after clicking the tab. New txs could be marked as well maybe.

Great idea

> Regarding using the "badgeText": Do you think there's a chance we might use this for something else in the future? Once a user is used to "these are new transactions" it might not be easy to use it for something else. Just a thought.

Even if there are other uses of badge, I don't see it as a problem. Badge can show 11 and I can have 10 new transactions and 1 other kind of notification.
